CHANGES.rst

```diff
@@ -1,6 +1,14 @@
 Changes
 =======
 
+Version 0.3.0 (2018-08-10)
+--------------------------
+
+- Adds support for Serial workflows.
```

Comment:
Super tiny thing before releasing, can you add a link to workflow engine serial here? I have done something similar in [reana-client release PR](https://github.com/reanahub/reana-client/pull/155/files#diff-db23dcd814354c954091a9b90dbfd92aR7).